frontend/PreferencesPanel.js

```diff
@@ -90,7 +90,7 @@ class PreferencesPanel extends React.Component {
               ref={this._setSelectRef}
               value={themeName}
             >
-              {!showHiddenThemes && (<option value="">{browserName}</option>)}
+              {browserName && (<option value="">{browserName}</option>)}
```

Comment:
> I'm still not sure I understand why someone would set a theme in the way shown in your gif above vs using the built-in theme setting. It seems...unnecessarily indirect.

Because the main usage of react-native-debugger is devtools, it should have an option similar to the browser extension have `match browser`.

It may just be a unilateral request, not sure if people need to go back to default theme for `react-devtools` package. (It had a `Reset` button but removed)

I reverted https://github.com/facebook/react-devtools/pull/778/commits/8f2c932528da76bf15954e5be6415a1cbab3fce8 because It may not be necessary.